libs/blockscout-chains/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use reqwest_middleware::ClientBuilder;
+use reqwest_retry::{policies::ExponentialBackoff, RetryTransientMiddleware};
+use serde::Deserialize;
+use std::collections::HashMap;
+
+const CHAINS_URL: &str = "https://chains.blockscout.com/api/chains";
+
+pub async fn get_blockscout_chains() -> anyhow::Result<BlockscoutChains> {
+    let max_retries = 3;
```

Comment:
nit: write builder to have access to max_retries and chains_url, like

```
let chains = BlockscoutChainsClient::default().fetch_all().await.unwrap()
```
or 
```
let chains = BlockscoutChainsClient::builder().with_max_retries(3).with_host(HOST).build().unwrap().fetch_all().await.unwrpa()
```

Reply:
Done